source/vistas/core/graphics/mesh.py

```diff
@@ -79,6 +79,13 @@ def __init__(
             glEnableVertexAttribArray(1)
             glVertexAttribPointer(1, 3, GL_FLOAT, GL_FALSE, sizeof(GLfloat) * 3, None)
 
+        if self.has_color_array:
```

Comment:
Any way to implement colors in the shader? Or does it depend on just way too much application context?